templates/humio@.service.j2

```diff
@@ -1,6 +1,8 @@
 [Unit]
   Description=Humio service
   After=network.service
+  StartLimitInterval=120
```

Comment:
Do you want StartLimitIntervalSec?

Reply:
It seems I was using an older form but it is still supported, will update.